### PR TITLE
New IA for DS

### PR DESCRIPTION
**Pull request by jmealing**
FR: https://github.com/canada-ca/systeme-conception/pull/336

Preview:

- https://deploy-preview-456--design-system-canada-ca.netlify.app/index.html
- https://deploy-preview-456--design-system-canada-ca.netlify.app/about/
- https://deploy-preview-456--design-system-canada-ca.netlify.app/specifications.html
- https://deploy-preview-456--design-system-canada-ca.netlify.app/specifications/information-findability
- https://deploy-preview-456--design-system-canada-ca.netlify.app/research-summaries/
- https://deploy-preview-456--design-system-canada-ca.netlify.app/specifications/usage-canadaca-design.html
- https://deploy-preview-456--design-system-canada-ca.netlify.app/specifications/mandatory-elements.html
- https://deploy-preview-456--design-system-canada-ca.netlify.app/specifications/information-findability/organizing-content.html
- https://deploy-preview-456--design-system-canada-ca.netlify.app/specifications/design-content.html

### CODE REVIEW

File: specifications.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    </div>
+    <div class="col-md-4">
+     <section>
+      <h3 class="h5">
+       <a href="./architecture/templates.html">
+        Designing content for Canada.ca
+       </a>
+      </h3>
+      <p>
+       Using evidence to design people-first and task-centred experiences.
```
Comment (donohuech) [inline specifications.md]: Can you please remove the period

File: specifications/information-findability.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+          {
+            "link": "https://design.canada.ca/specifications.html",
+            "title": "Canada.ca Specifications"
+          }
+        ]
+---
+
+
+<p>
+ Organize and structure webpages for findability and task success across Canada.ca.
```
Comment (donohuech) [inline specifications/information-findability.md]: Can we please make 'webpages' two words. So 'web pages'

**Review by donohuech — COMMENTED** [review]
@jmealing only two small changes- both my fault for not catching in the original Word docs

**Comment by jmealing** [comment]
@donohuech added the changes from this document too: 
https://docs.google.com/document/d/1R_07yyZhwvuXnc2Z4EgQfIE0mT3RGpVr/edit

File: research-summaries/index.md [hunk 3]
```diff
@@ -29,3 +29,8 @@ title: "Research summaries"
   <li><a href="/research-summaries/weather-research-summary.html">Weather optimization</a></li>
   <li><a href="/research-summaries/diseases-research-summary.html">Diseases and immunization</a></li>
 </ul>
+<h2>Past projects with our partners</h2>
+<p>Learn about work we’ve done with other departments and organizations to improve task success on Canada.ca:</p>
+<ul>
+  <li><a href="./partners/">Working with partners</a></li>
```
Comment (donohuech) [inline research-summaries/index.md]: @jmealing this link isn't working

**Review by donohuech — APPROVED** [review]

**Pull request merged**